packages/zoe/src/contractSupport/ratio.js

```diff
@@ -233,16 +223,10 @@ export const addRatios = (left, right) => {
 export const subtractRatios = (left, right) => {
   assertIsRatio(right);
   assertIsRatio(left);
-  assert.equal(
-    left.numerator.brand,
-    right.numerator.brand,
-    X`numerator brands must match: ${q(left)} ${q(right)}`,
-  );
-  assert.equal(
-    left.denominator.brand,
-    right.denominator.brand,
-    X`denominator brands must match: ${q(left)} ${q(right)}`,
-  );
+  left.numerator.brand === right.numerator.brand ||
+    Fail`numerator brands must match: ${q(left)} ${q(right)}`;
+  left.denominator.brand === right.denominator.brand ||
+    Fail`denominator brands must match: ${q(left)} ${q(right)}`;
```

Comment:
It would be nice if there was more consistency for such messages (packages/time/src/timeMath.js uses `${q(leftBrand)} vs ${q(rightBrand)}` while packages/zoe/src/contractSupport/ratio.js uses `${q(left)} ${q(right)}` with no "vs"), but I think that's out of scope for this PR.

Reply:
Agreed. But also agreed out of scope. Resolving.